pkg/client/unversioned/scheduledjobs_test.go

```diff
@@ -0,0 +1,277 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unversioned_test
+
+import (
+	"testing"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/testapi"
+	"k8s.io/kubernetes/pkg/apis/batch"
+	"k8s.io/kubernetes/pkg/apis/batch/v2alpha1"
+	"k8s.io/kubernetes/pkg/client/unversioned/testclient/simple"
+)
+
+func getScheduledJobsResource() string {
+	return "scheduledjobs"
+}
+
+func TestListScheduledJob(t *testing.T) {
```

Comment:
@soltysh is this test similar to other tests in the folder, which verify that various client methods work? If so, then it might not be necessary, becasue the generated clientset test all the methods. We plan to deprecate the pkg/client/unversioned and ask people to use clientset in the near future. It's just a FYI, we can still have these tests merged.